app/models/queue_item.rb

```diff
@@ -3,7 +3,7 @@ class QueueItem < ActiveRecord::Base
   belongs_to :video
 
   delegate :category, to: :video
-  delegate :title, to: :video, prefix: :video
+  delegate(:title, {:to => :video, :prefix => :video})
```

Comment:
This syntax is not preferred in Ruby community. Its called old hash rocket style and was used up to Ruby 1.9.

If you want to keep brackets around then `delegate(:title, { to::video, prefix: :video })`